app/scripts/controllers/SysinfoController.coffee

```diff
@@ -103,5 +105,16 @@ angular.module('neo4jApp.controllers')
       $scope.autoRefresh = !$scope.autoRefresh
       refreshLater()
 
+    $scope.isCurrentConnection = (addresses) ->
+      if Settings.useBolt
+        boltHost = Settings.boltHost || $rootScope.boltHost
+        boltAddressForMember = Utils.getServerAddressByProtocol('bolt', addresses)[0]
+        return boltAddressForMember && boltAddressForMember.includes(boltHost)
```

Comment:
This `includes` will always be true if you're running a cluster on the same host/ip and don't have a port specified in your bolt host settings.

![oskar4j 2016-10-04 at 11 24 40](https://cloud.githubusercontent.com/assets/570998/19068999/2f0d82c2-8a25-11e6-92f0-6e60c37c0344.png)